jujugui/static/gui/src/app/components/machine-view/machine-view.js

```diff
@@ -19,40 +19,42 @@ with this program.  If not, see <http://www.gnu.org/licenses/>.
 'use strict';
 
 class MachineView extends React.Component {
-  constructor(props) {
-    super(props);
+  constructor() {
+    super();
```

Comment:
Doesn't the constructor also set up this.props? From https://facebook.github.io/react/docs/state-and-lifecycle.html:
"Class components should always call the base constructor with props". Surely it does not hurt, and we keep the same signature when overriding the constructor?